docs/source/cognite.rst

```diff
@@ -2,13 +2,28 @@ Quickstart
 ==========
 Authenticate
 ------------
-The preferred way to authenticate against the Cognite API is by setting the :code:`COGNITE_API_KEY` environment variable. All examples in this documentation require that the variable has been set.
+
+The preferred way to authenticating against the Cognite API is using native tokens. To enable this, the CogniteClient
```

Comment:
native tokens is only an internal Cognite thing, it's should be named OpenID Connect (OIDC)